applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -36,6 +36,9 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
 
         elif solver_type == "two_fluids" or solver_type == "TwoFluids":
             solver_module_name = "navier_stokes_two_fluids_solver"
+        
+        elif solver_type == "two_fluids_hydraulic" or solver_type == "TwoFluidsHydraulic":
```

Comment:
```suggestion
        elif solver_type == "two_fluids_hydraulic":
```

please follow the style guide regarding naming for new additions

Reply:
I think the correct name should be "two_fluid_hydraulic"

Reply:
Done!